packages/dds/merge-tree/src/localReference.ts

```diff
@@ -283,7 +283,7 @@ export class LocalReferenceCollection {
 					if (next.done === true) {
 						subiterators.shift();
 					} else {
-						return { done: next.done, value: next.value.data };
+						return { done: next.done as boolean, value: next.value.data };
```

Comment:
The current implementation is technically incorrect, as the property should either be explicitly false or omitted entirely to be compliant.  Suggest

```suggestion
						return { done: false, value: next.value.data };
```

Reply:
https://tc39.es/ecma262/multipage/control-abstraction-objects.html#table-iteratorresult-interface-properties